scripts/pin-to-cluster.sh

```diff
@@ -58,7 +58,7 @@ root_cid=$(ipfs-cluster-ctl \
     --name "$PIN_NAME" \
     --recursive "$INPUT_DIR" ) || {
   # If it fails, show the ipfs-cluster-ctl command and the error message
-  echo "ipfs-cluster-ctl --host $HOST --basic-auth *** add --quieter --local --cid-version 1 --name '$PIN_NAME' --recursive $INPUT_DIR" 1>&2
+  echo "ipfs-cluster-ctl --host $HOST --basic-auth *** add --wait --quieter --local --cid-version 1 --name '$PIN_NAME' --recursive $INPUT_DIR" 1>&2
```

Comment:
can you confirm that `add` supports `--wait` ? IIRC only `pin add` does.

Reply:
im also noticing I edited the printed error message

Reply:
https://github.com/ipfs/ipfs-cluster/issues/1285